index.bs

```diff
@@ -1341,7 +1341,9 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
         Note: An [=RP ID=] is based on a [=host=]'s [=domain=] name. It does not itself include a [=scheme=] or [=port=], as an [=origin=] does. The [=RP ID=] of a [=public key credential=] determines its <dfn>scope</dfn>. I.e., it <dfn>determines the set of origins on which the public key credential may be exercised</dfn>, as follows:
 
             - The [=RP ID=] must be equal to the [=determines the set of origins on which the public key credential may be exercised|origin=]'s [=effective domain=], or a [=is a registrable domain suffix of or is equal to|registrable domain suffix=] of the [=determines the set of origins on which the public key credential may be exercised|origin=]'s [=effective domain=].
-            - The [=determines the set of origins on which the public key credential may be exercised|origin=]'s [=scheme=] must be `https`.
+            - One of the following is true:
+                - The [=determines the set of origins on which the public key credential may be exercised|origin=]'s [=scheme=] is `https`.
+                - The [=determines the set of origins on which the public key credential may be exercised|origin=] is `localhost` and the [=scheme=] is `http`.
```

Comment:
On second thought - to be precise, shouldn't this be "The origin's host is `localhost` and its scheme is `http`"? Something like:

```suggestion
            - One of the following must be true:
                - The [=determines the set of origins on which the public key credential may be exercised|origin=]'s [=scheme=] is `https`.
                - The [=determines the set of origins on which the public key credential may be exercised|origin=]'s [=host=] is `localhost` and its [=scheme=] is `http`.
```

(I haven't checked if `[=host=]` is a valid autolink; please check that before committing this)

Reply:
@emlun autolink from [=host=] goes to: https://url.spec.whatwg.org/#concept-url-host

Reply:
Ok, I think we should be using [`[=origin/scheme=]`](https://html.spec.whatwg.org/multipage/browsers.html#concept-origin-scheme) and [`[=origin/host=]`](https://html.spec.whatwg.org/multipage/browsers.html#concept-origin-host) then.